content/applications/inventory_and_mrp/manufacturing/workflows/production_analysis.rst

```diff
@@ -0,0 +1,320 @@
+:guilabel:`Measures` :icon:`fa-caret-down` button, and selecting one of the following options from
+the drop-down menu:
+
+- :guilabel:`Average Component Cost/Unit`: the average cost of the components required to produce
+  one unit of the product.
+- :guilabel:`Average Employee Cost/Unit`: the average cost paid to employees to produce one unit of
+  the product.
+- :guilabel:`Average Subcontracting Cost/Unit`: the average cost of engaging a subcontractor to
+  produce one unit of the product.
+- :guilabel:`Average Total Cost/Unit`: the average cost of producing one unit of the product,
+  including component, employee, operation, and subcontracting costs.
+- :guilabel:`By-Products Total Cost`: the total value of all by-products created by manufacturing
+  the product.
+- :guilabel:`Duration of Operations/Unit`: the average total duration of operations required to
+  produce one unit of the product.
+- :guilabel:`Quantity Demanded`: the total number of units of the product included in |MOs|.
+- :guilabel:`Quantity Produced`: the total number of units of the product that have actually been
+  produced.
+- :guilabel:`Total Component Cost`: the total amount spent on the product's components, across every
+  MO for the product.
+- :guilabel:`Total Cost`: the total amount spent manufacturing every unit of the product produced so
+  far.
+- :guilabel:`Total Duration of Operations`: the cumulative duration of every operation completed
+  while manufacturing the product.
+- :guilabel:`Total Employee Cost`: the cumulative amount paid to employees to manufacture the
+  product.
+- :guilabel:`Total Operation Cost`: the cumulative amount spent on operations required to produce
+  the product.
+- :guilabel:`Total Operation Cost/Unit`: the average cost of the operations required to produce
+  one unit of the product.
+- :guilabel:`Total Subcontracting Cost`: the cumulative amount paid to subcontractors to produce the
+  product.
+- :guilabel:`Yield Percentage (%)`: the total quantity of the product produced versus the total
+  quantity demanded, represented as a percentage.
+- :guilabel:`Count`: the total count of |MOs| created for the product.
+
+.. note::
+   Only one measure can be selected at a time.
+
+Views
+=====
+
+By default, the production analysis dashboard displays data using the *line chart* graph type of the
+*graph view*.
+
+To select the bar chart or pie chart graph type instead, click the :icon:`fa-bar-chart`
+:guilabel:`(bar chart)`, or :icon:`fa-pie-chart` :guilabel:`(pie chart)` button below the
+:guilabel:`Search...` bar. To return to the line chart, click the :icon:`fa-line-chart`
+:guilabel:`line chart` button.
+
+To see production data displayed in a pivot table instead of a graph, click the
+:icon:`oi-view-pivot` :guilabel:`(pivot view)`, button to the right of the :guilabel:`Search...`
+bar. To return to graph view, click the :icon:`fa-area-chart` :guilabel:`(graph view)` button.
+
+Bar chart
+---------
```

Comment:
might be cool to use anchors / ref links to these views, so the reader can quickly click-and-access them throughout the doc. Not required, just a suggestion to consider...